aggregators/can-load-offline/index.js

```diff
@@ -19,14 +19,25 @@
 
 const Aggregate = require('../aggregate');
 
+/**
+ * @type {string}
+ */
+const serviceWorker = require('../../audits/offline/service-worker').name;
```

Comment:
oh right, this may just be from what you mentioned about requires needing to be at the top level. It would be interesting to see if that's because it uses the same underlying system as ES2015 modules